src/js/workshop-comments.js

```diff
@@ -124,15 +124,19 @@ function disableFields(disabled) {
 	// When a UIO theme is selected, the background color will remain the same as what is set by the theme.
 	const backgroundColor = disabled ? "#f3f3f3" : "#fff";
 
-	// Disable/enable and set background color of post a comment button
+	// Disable/enable and set background color of post a comment button.
 	const postCommentButton = document.getElementById("post-comment");
 	postCommentButton.setAttribute("style", "background-color: " + backgroundColor);
 	postCommentButton.disabled = disabled;
+	// When UIO themes are turned on the opacity of the post a commnet button will change while disabled instead of background color.
```

Comment:
1. "When a UIO theme is turned on, ...": add comma;
2. Same as the previous comment, add quotes around "post a comment".

Please make sure all your comments still make sense when you or other developers look at them after some time.